app/src/main/res/values/dimens.xml

```diff
@@ -57,8 +57,6 @@
     <!-- We want to keep button_size and urlbar height the same to keep them aligned -->
     <dimen name="urlbar_height">@dimen/button_size</dimen>
 
-    <!-- In HomeTileScreenshotStore, we generate screenshots of this size and scale them:
-         it should be a fine estimate for now. -->
-    <dimen name="home_tile_icon_estimated_width">480px</dimen>
-    <dimen name="home_tile_icon_estimated_height">270px</dimen>
+    <!-- In HomeTileScreenshotStore, we generate placeholder screenshot icons of this size -->
```

Comment:
nit: I think this is redundant to the name: I think you can remove it.

I had the comment before to emphasize that we're making an estimate which is scaled to the right size, rather than giving an exact value (which is how I'd expect to use it).